src/designer/elsa-workflows-studio/src/modules/credential-manager/services/credential-manager.client.ts

```diff
@@ -0,0 +1,67 @@
+import axios, {AxiosInstance, AxiosRequestConfig} from "axios";
```

Comment:
I don't think we should copy the entire elsa client ts file. Surely we can reuse the existing one?